src/constants.js

```diff
@@ -15,4 +19,16 @@ export const QUESTION_TYPES = {
     eslintConfig: 'eslintConfig',
     polyfill: 'polyfill',
     styledComponents: 'styledComponents',
+};
+
+export const PROMISIFIED_METHODS = {
```

Comment:
liever `fs-extra` gebruiken (zit er al in denk?, weet niet meer precies waarom ik dat niet in de 1e plaats overal gebruikt heb)